src/App.js

```diff
@@ -1,25 +1,41 @@
-import React from "react";
-import logo from "./logo.svg";
+import React, { useState } from "react";
+import { Switch, Route, BrowserRouter } from "react-router-dom";
 import "./App.css";
+import List from "./List/List";
+import AddItem from "./AddItem/AddItem";
+import Footer from "./Footer/Footer";
+import Header from "./Header/Header";
 
 function App() {
+  const [isListSelected, setIsListSelected] = useState(true);
+  const [isAddSelected, setIsAddSelected] = useState(false);
+
+  const selectListHandler = () => {
+    setIsListSelected(true);
+    setIsAddSelected(false);
+  };
+
+  const selectAddHandler = () => {
+    setIsListSelected(false);
+    setIsAddSelected(true);
+  };
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Header isAddSelected={isAddSelected} onListSelect={selectListHandler} />
+      <div className="App">
+        <Switch>
+          <Route path="/" exact render={() => <List />} />
```

Comment:
One really cool thing you can do here is this: `<Route render={List} />`

Since components are just functions, you can pass them in like that and it'll render correctly! Routes pass in lots of variables that can be useful to the child component, and you may want them down the road.

Reply:
But you don't mean we should remove the path or exact, right?

Reply:
Updated. :-)

Reply:
Now I see it! Nevermind! Looks way better that way and totally makes sense.